glusterd2/snapshot/lvm/lvm.go

```diff
@@ -4,21 +4,37 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"strconv"
 	"strings"
 
 	"github.com/gluster/glusterd2/glusterd2/brick"
 	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/pkg/api"
+
 	"github.com/pborman/uuid"
-	"github.com/sirupsen/logrus"
 )
 
-var (
-	//CreateCommand is path to lvm create
-	CreateCommand = "/sbin/lvcreate"
-	//RemoveCommand is path to lvm create
-	RemoveCommand = "/sbin/lvremove"
+const (
+	//CreateCommand is path to lvcreate
+	CreateCommand string = "/sbin/lvcreate"
```

Comment:
Use `exec.LookupPath` to set these. You may need to change them to be variables.